frontend/src/components/Chat.tsx

```diff
@@ -1,20 +1,39 @@
 import { useState, useEffect, useRef } from 'react';
+import { useHistory } from 'react-router-dom';
```

Comment:
Jak to jest ze spacjami przy nawiasach {}? Powinno być {useHistory} czy { useHistory }? Wyżej w sessions.ts jest bez spacji

Reply:
jestem przyzwyczajony do nie robienia odstępów, ale mój obecny vscode to formatuje z odstępami xD
Trzeba to rozstrzygnąć na czacie

Reply:
na froncie narazie zostawię, na backendzie sprawdzę i uspójnię do braku spacji